index.js

```diff
@@ -33,4 +25,37 @@ module.exports = function (func, cb) {
   } catch (e) {
     setImmediate(cb.bind(null, e));
   }
+}
+
+/**
+ * Run a function asynchronously or synchronously
+ * @param   {Function} func  Function to run
+ * @param   {Function} cb    Callback function passed the `func` returned value
+ * @...rest {Mixed}    rest  Arguments to pass to `func`
+ * @return  {undefined}
+ */
```

Comment:
I moved the exports down to the bottom.

Seemed like it would be best to keep the important bit of code at the top. I am not married to the idea.